kernels/volk/volk_32fc_s32fc_x2_rotator_32fc.h

```diff
@@ -432,103 +432,64 @@ static inline void volk_32fc_s32fc_x2_rotator_32fc_a_avx(lv_32fc_t* outVector, c
         incr *= (phase_inc);
     }
 
-    /*printf("%f, %f\n", lv_creal(phase_Ptr[0]), lv_cimag(phase_Ptr[0]));
-    printf("%f, %f\n", lv_creal(phase_Ptr[1]), lv_cimag(phase_Ptr[1]));
-    printf("%f, %f\n", lv_creal(phase_Ptr[2]), lv_cimag(phase_Ptr[2]));
-    printf("%f, %f\n", lv_creal(phase_Ptr[3]), lv_cimag(phase_Ptr[3]));
-    printf("incr: %f, %f\n", lv_creal(incr), lv_cimag(incr));*/
-    __m256 aVal, phase_Val, inc_Val, yl, yh, tmp1, tmp2, z, ylp, yhp, tmp1p, tmp2p;
+    __m256 aVal, phase_Val, z;
 
     phase_Val = _mm256_loadu_ps((float*)phase_Ptr);
-    inc_Val = _mm256_set_ps(lv_cimag(incr), lv_creal(incr),lv_cimag(incr), lv_creal(incr),lv_cimag(incr), lv_creal(incr),lv_cimag(incr), lv_creal(incr));
-    const unsigned int fourthPoints = num_points / 4;
+    
+    const __m256 inc_Val = _mm256_set_ps(lv_cimag(incr), lv_creal(incr),
+                                         lv_cimag(incr), lv_creal(incr),
+                                         lv_cimag(incr), lv_creal(incr),
+                                         lv_cimag(incr), lv_creal(incr));
```

Comment:
``_mm256_castpd_ps(_mm256_broadcast_sd((const double*)&incr))`` (see e.g. ``volk_32fc_x2_s32f_square_dist_scalar_mult_32f_a_avx()``). On the other hand, the compiler probably figures this out.

Also applies to the unaligned version.

Reply:
I didn't change the implementation here. But I like this solution because it is verbose without any assumptions. Thus, I'd prefer to leave it as is.